multidim-interop/impl/rust/v0.52/Makefile

```
@@ -1,23 +1,35 @@
 image_name := rust-v0.52
-commitSha := 68e6bf9c3cd0d3317415a5ba31a62e91cba0a5d2
+version := 0.52.1
```

```suggestion
version := 0.52.2
```

I released `libp2p` `v0.52.2` today. Want to include it in this pull request?

https://github.com/libp2p/rust-libp2p/pull/4312

No worries in case you don't want to mix these efforts in the same PR. I am happy to do a follow-up pull request.

---

I'll include 0.52.2 in here 👍